_targets/build.project.armv7a7-imx6ull

```diff
@@ -29,51 +29,60 @@ export PORTS_OPENVPN=y
 export PORTS_JANSSON=y
 export PORTS_CURL=y
 
+#
+# Boot configuration (project specific)
```

Comment:
yeah, it should be in project build config, not in generic `TARGET_SUBFAMILY` one.

This file is also used by the DTR project so for now booting from NAND (with host syspage generation) needs to be retained.

The decision to use NOR/PLO as a booting device needs to be done in `_projects/armv7a7-imx6ull-evk/build.project` (but this file should probably support both options or keep only the common part of the booting procedure).

I'm voting **request changes** to avoid accidental merge of this PR before restoring NAND booting functionality. Please also test it on HW before the merge ;).